tests/resources/files/apps/maven/liberty-maven-test-app/pom.xml

```diff
@@ -42,9 +42,9 @@
             <scope>test</scope>
```

Comment:
Why is there a separate test/resources/files dir with pom and build.gradle ?  I remember there being a 'files' dir to add in the mvnw.